src/main/java/frc/team2412/robot/commands/launcher/SetAngleCommand.java

```diff
@@ -17,6 +17,8 @@ public SetAngleCommand(LauncherSubsystem launcherSubsystem, double angle) {
 
 	@Override
 	public void initialize() {
+		System.out.println("Setting Angle!");
```

Comment:
Should revert this, we log when commands are initialized [here](https://github.com/robototes/Crescendo2024/blob/798f7938ddbb0e3b074043232ab81e3c1fca8023/src/main/java/frc/team2412/robot/Robot.java#L83).

Reply:
done